examples/visualization/styles/helpers/size-categories-style.html

```diff
@@ -37,28 +37,28 @@
 
     <script>
       async function initialize() {
-        carto.setDefaultCredentials({ username: 'public' });
+        carto.setDefaultCredentials({ username: 'cartoframes' });
```

Comment:
I suppose you've committed the example by mistake, but I prefer to have the point examples because is more common to use this helper with a point layer rather than lines.

Reply:
Not, I did it on purpose. There is no examples with lines and also, this one is the one used in CF for the very same helper

Reply:
Ok!